include/l3.h

```diff
@@ -36,6 +45,17 @@
  */
 int l3_init(const char *path);
 
+/**
+ * \brief Caller-macro to invoke L3 simple logging.
+ */
+#ifdef L3_LOC_ENABLED
+#define l3_log_simple(msg, arg1, arg2)                          \
+        l3__log_simple(__LOC__, (msg), (arg1), (arg2))
```

Comment:
In an immediate future, if we build the sample programs with `L3_LOC_ENABLED=1`, then we will store the LOC-ID token in the L3 logging entry struct. Otherwise, will just store 0.